packages/ui/src/hooks/useMarketplaces.ts

```diff
@@ -53,14 +54,23 @@ export default function (
             marketplace.imageUrl = data.icon
           }
         }
+        if (marketplace.orderbook === 'opensea') {
+          const osFee =
+            royaltyBps && royaltyBps > 50 ? 0 : 50 - (royaltyBps || 0)
```

Comment:
Should this be greater than or equal to 50? `royaltyBps >= 50`

Reply:
Oh you're right, nice catch!

Reply:
Fixed!